samples/winapi_drivelist/main.c

```diff
@@ -56,11 +57,25 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
 
-    while(1) {
-        Sleep(2000);
+    debugPrint("\nDone!");
+
+cleanup:
+    ret = nxUnmountDrive('E');
+    if (!ret) {
+        error = GetLastError();
+        debugPrint("\nFailed to unmount E: drive! Error code: %x\n", error);
+    }
+
+unmount_c:
+    ret = nxUnmountDrive('C');
+    if (!ret) {
+        error = GetLastError();
+        debugPrint("\nFailed to unmount C: drive! Error code: %x\n", error);
     }
 
-    return 0;
```

Comment:
Why the removal? `main` returns an int, so there should be a return statement, even if it's never reached.